<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadWindow</class>
 <widget class="QDialog" name="ReadWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Read Window[Room1]</string>
  </property>
  <widget class="QPushButton" name="pushButton_refresh">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>390</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>350</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>5</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>282</width>
     <height>324</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Photo resistor voltage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Photo Resistor Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Light State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Room Temperture</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Temperature Reference</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Fan State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Heater State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Gas Voltage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Gas Dangerous Level</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Fire Voltage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Fire State</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_35">
        <property name="text">
         <string>Water Leakage State</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_photo_resistor_volt">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_photo_resistor_ref">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_light_state">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_room_temp">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_temp_ref">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fan_state">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_heater_state">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_gas_volt">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_gas_danger_level">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fire_volt">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_fire_state">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_water_state">
          <property name="text">
           <string>.................</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_v_gas">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
